src/components/UserInviteForm.tsx

```diff
@@ -89,14 +89,18 @@ export const UserInviteForm: React.FC<UserInviteFormProps> = ({ card, onCreate }
             <FormActions flat="top">
                 <FormAction left />
                 <FormAction right inline>
-                    <Button size="l" view="primary" type="submit" disabled={!isValid} text={t('Send invite')} />
+                    {/*
+ // @ts-ignore */}
```

Comment:
Add #25 to every of ts-ignore comments plz. It must be fixed before production.

Reply:
done